src/views/course-edit.vue

```diff
@@ -54,6 +54,10 @@ Dependencies:
 
       <course-change-category @changedCategory="storeCourse"></course-change-category>
 
+      <course-feedback @success="$bvToast.show('author-toast')"></course-feedback>
```

Comment:
the placement seems odd. Wouldn't it be better to have the option at the button, alongside course preferences?